src/game_timer.cpp

```diff
@@ -0,0 +1,52 @@
+// Header
+#include "game_timer.hpp"
+
+// stlib
+#include <vector>
+#include <algorithm>
+#include <sstream>
+
+bool GameTimer::init(float world_scale)
+{
+	struct tm init_time = {0};
```

Comment:
can you explain what this struct is? Is it some `ctime` thing?

Reply:
Yep! http://www.cplusplus.com/reference/ctime/tm/